src/items/equipment/polyclub_advanced.json

```diff
@@ -12,26 +12,48 @@
     "actionTarget": "kac",
     "actionType": "mwak",
     "activation": {
-      "type": "none",
+      "type": "action",
```

Comment:
Why do these all have activation of action now?

Reply:
Guess this is a point of determining our standard here. At least Ranged weapons currently have to have activation in order to list range. So should we, or should we not, have that set for melee weapons? Either way, we have melee weapons going back and forth on this.
I suppose for ranged, the button on the sheet is hidden. So your call.

Reply:
oh wait, I see. the guns are all none. which doesn't show a button for weapons. alright, I'll change it back.